src/editor/logic.py

```diff
@@ -364,7 +364,10 @@ def send_review_request(
         attachment=None,
         access_key=None,
 ):
-    from_email = get_setting('from_address', 'email')
+    from_email = sender.email
```

Comment:
`from_email = sender.email or get_setting('from_address', 'email')` ?

A one liner would make it much easier to read, especially because we are replicating this over and over again

Reply:
I agree, or you could do `from_email = sender.email if sender.email else get_setting('from_address', 'email')`

but I like @fradeve suggestion better